src/FSharp.Markdown/MarkdownParser.fs

```diff
@@ -449,6 +449,35 @@ let rec (|ListItems|_|) prevSimple = function
 
 // Code for parsing pipe tables
 
+// Splits table row into deliminated parts escaping inline code and latex
+let rec PipeTableFindSplits (delim : char array) (line : char list) = 
```

Comment:
No problem. Do I create a new pull request or what?

On 03/04/2016 16:19, Matthias Dittrich wrote:
>
> In src/FSharp.Markdown/MarkdownParser.fs 
> <https://github.com/tpetricek/FSharp.Formatting/pull/394#discussion_r58311801>:
>
> > @@ -449,6 +449,35 @@ let rec (|ListItems|_|) prevSimple = function
> >
> >  // Code for parsing pipe tables
> >
> > +// Splits table row into deliminated parts escaping inline code and latex
> > +let rec PipeTableFindSplits (delim : char array) (line : char list) =
>
> Its more about the code-style: The function is fine but please call it 
> "pipeTableFindSplits" instead of "PipeTableFindSplits". Only active 
> pattern are a exception here.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/tpetricek/FSharp.Formatting/pull/394/files/1fbfa6f43f20dd312d7459c6708ad2b1a12047b9#r58311801>
>